sigs.yaml

```diff
@@ -74,6 +74,22 @@ sigs:
     - dir: sig-storage
       label: sig/storage
       name: storage
+      leadership:
+        chairs:
+          - github: alicefr
+            name: "Alice Frosi"
+            company: Red Hat
+          - github: mhenriks
+            name: "Michael Henriksen"
+            company: Red Hat
+      meetings:
+        - description: KubeVirt Storage SIG Meeting
+          day: Thursday
```

Comment:
Still Thursday, where it should be Monday: https://calendar.google.com/calendar/u/0/embed?src=kubevirt@cncf.io